gui/screens/settings.py

```diff
@@ -5,6 +5,7 @@
 
 import configs.timeouts
 import driver
+from objectmaphelper import RegularExpression
```

Comment:
I think we should put this import at the very top

Reply:
done